### DISCUSSION TITLE
Show play button when video is paused

### DISCUSSION THREAD

**Discussion by Scippy** [videojs/video.js#8606] [General]
Hi, I need to show the play button above the video when is paused.
When I load the page I can see the play button, then when I click it rightly disappears, but I would like it to reappear when the video is clicked and paused.
how can I do?
Thanks!

**Comment by amtins** [comment]
Hey @Scippy depending on how you initialize the player, you can do the following:

```html
<!-- With the HTML tag -->
<video id="player" class="video-js vjs-show-big-play-button-on-pause" controls></video>
```

```html
<!-- With the video-js element -->
<video-js id="player" class="vjs-show-big-play-button-on-pause" controls></video-js>
```

Code reference:
https://github.com/videojs/video.js/blob/a57b07a418636daf9723a28715f96f2e3152ddc3/src/css/components/_big-play.scss#L59-L60

**Reply by Scippy** [reply]
Great! Very thanks @amtins !
